apps/schools/domains/query/components/queryList/index.tsx

```diff
@@ -199,15 +193,21 @@ export function QueryList() {
                         },
                     },
                     circle_name: {
-                        filteredValue: Array.from(circleFilter),
+                        filteredValue: circles ?? [],
                     },
                 }}
                 sortFields={['created_at']}
                 searchRequestText={searchRequestText}
                 setSearchRequestText={setSearchRequestText}
                 onChange={(pagination, filters, sorter) => {
-                    setCircleFilter(filters['circle_name'] ?? [])
-                    setStatusFilter((filters['status'] ?? []).join(','))
+                    const localStatuses = [...(filters['status'] ?? [])] as string[];
+                    setStatuses(
+                        localStatuses.length === 0 ? null : localStatuses
+                    );
+                    const localCircles = [...(filters['circle_name'] ?? [])] as string[];
+                    setCircles(
+                        localCircles.length === 0 ? null : localCircles
+                    );
```

Comment:
need to think about where to use it?? and where || because ?? skips values such as 0, an empty string, an empty array, etc. as true.

Reply:
usually we need to avoid undefined values so mostly ?? is acceptable

Reply:
|| avoid undefined too